### DISCUSSION TITLE
Syntax trouble with cue_cut function

### DISCUSSION THREAD

**Discussion by Flopblopper** [savonet/liquidsoap#2516] [Q&A] [answered]
Hello,

I'm trying to skip the first and last two minutes of every file in my playlist. I'm using these two lines:

```
audio = playlist(prefix="annotate:liq_cue_in=\"120\",liq_cue_out=\"120\":",prefetch=4,loop=true,mode="normal", "/home/liquidsoap/Audio")
audio = cue_cut(cue_in_metadata=liq_cue_in,cue_out_metadata=liq_cue_out, audio)
```

The problem is that the two lic_cue variables are not being defined, so I assume that I'm doing something incorrectly. I was hoping that one of you may be able to spot the error.

Thanks.

**Comment by smimram** [comment, marked as answer]
You need strings here I think (note the quotes):
```
audio = cue_cut(cue_in_metadata="liq_cue_in",cue_out_metadata="liq_cue_out", audio)
```

**Reply by Flopblopper** [reply]
Thanks, that worked perfectly.

Now I just wish that I had read about cue_cut more carefully. It seems that cue_out is relative to the beginning of the file instead of the end like I had originally thought.
